src/Bicep.Core/Features/FeatureProvider.cs

```diff
@@ -42,6 +42,8 @@ public FeatureProvider(RootConfiguration configuration)
 
         public static TraceVerbosity TracingVerbosity => ReadEnumEnvvar("BICEP_TRACING_VERBOSITY", TraceVerbosity.Basic);
 
+        public bool DynamicTypeLoading => configuration.ExperimentalFeaturesEnabled.DynamicTypeLoadingEnabled;
```

Comment:
```suggestion
        public bool DynamicTypeLoadingEnabled => configuration.ExperimentalFeaturesEnabled.DynamicTypeLoading;
```

Reply:
If I follow the advise in https://github.com/Azure/bicep/pull/10868#discussion_r1267313128c then it should stay without change, that is:
`public bool DynamicTypeLoading => configuration.ExperimentalFeaturesEnabled.DynamicTypeLoading;`